zilliqa/src/time.rs

```diff
@@ -8,6 +8,7 @@ pub type SystemTime = std::time::SystemTime;
 
 pub type OffsetDateTime = time::OffsetDateTime;
 
+#[cfg(feature = "fake_time")]
```

Comment:
This is needed in order to compile on my system sometimes. Not entirely sure why

Reply:
It should be unnecessary now that #563 is merged.